src/current/v23.2/upgrade-cockroach-version.md

```diff
@@ -193,6 +189,8 @@ These steps perform an upgrade to the latest {{ page.version.version }} release,
     cp -i {COCKROACHDB_DIR}/cockroach /usr/local/bin/cockroach
     ~~~
 
+1. If a cluster has corrupt descriptors, a major-version upgrade cannot be finalized. In CockroachDB v23.2 and above, automatic descriptor repair is enabled by default. After restarting each cluster node on v23.2, monitor the cluster logs for errors. If a descriptor cannot be repaired automatically, [contact support](https://support.cockroachlabs.com/hc) for assistance completing the upgrade. To disable automatic descriptor repair (not generally recommended), set the environment variable `COCKROACH_RUN_FIRST_UPGRADE_PRECONDITION` to `false`.
```

Comment:
same comment as above re: linking to logging docs

"not generally recommended" could be shortened to "not recommended"

Reply:
the link to 'support.cockroachlabs.com' doesn't seem to require the 'hc' at the end; it does redirect to that but I'd recommend leaving it off so we don't have a broken link if the support team changes their system and/or redirect in the future